backend-go/api/codepair/v1/models/model_create_upload_presigned_url_response.go

```
@@ -0,0 +1,10 @@
+package models
+
+type CreateUploadPresignedUrlResponse struct {
```

golangci-lint (1.62.2)

[warning] var-naming: type CreateUploadPresignedUrlResponse should be CreateUploadPresignedURLResponse

(revive)